run2-ultra-legacy-production/ultra_processing/click/testing.py

```diff
@@ -0,0 +1,374 @@
+import os
```

Comment:
It would be good to remove `click` sources from here and simply rely on it being installed in a a virtual environment...